app/viewmodels/SummaryCardHelper.scala

```diff
@@ -179,10 +179,18 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
           )
         )
       ),
-      summaryListRow(messages("summaryCard.lpp.key2"), Html(messages("summaryCard.lpp.15days")))
+      getPenaltyReason(lpp)
     )
   }
 
+  private def getPenaltyReason(lpp: LatePaymentPenalty)(implicit messages: Messages):SummaryListRow={
+    (lpp.`type`,lpp.reason) match{
+      case(PenaltyTypeEnum.Additional,"VAT_NOT_PAID_ON_TIME") =>
+        summaryListRow(messages("summaryCard.lpp.key2"), Html(messages("summaryCard.lpp.additional.30days")))
+      case(_,_) => summaryListRow(messages("summaryCard.lpp.key2"), Html(messages("summaryCard.lpp.15days")))
+    }
+  }
+
```

Comment:
Slight refactor to remove code duplication + swapped messages to correct reason
```suggestion
      summaryListRow(messages("summaryCard.lpp.key2"), Html(messages(getPenaltyReasonKey(lpp))))
    )
  }

  private def getPenaltyReasonKey(lpp: LatePaymentPenalty):String={
    lpp.reason match{
      case "VAT_NOT_PAID_ON_TIME" => "summaryCard.lpp.15days"
      case "VAT_OVERDUE_BY_31_DAYS" || "VAT_NOT_PAID_WITHIN_31_DAYS" => "summaryCard.lpp.additional.30days"
      case _ => "summaryCard.lpp.additional.30days" //TODO Add error handling + logging for unexpected reason
    }
  }


```
Will require test updates + would be good add at least add log in if not all error handling